src/voice-broadcast/models/VoiceBroadcastRecording.ts

```diff
@@ -268,7 +268,7 @@ export class VoiceBroadcastRecording
             event_id: this.infoEvent.getId(),
         };
         content["io.element.voice_broadcast_chunk"] = {
-            sequence: this.sequence++,
+            sequence: ++this.sequence,
```

Comment:
Maybe adding a bit of context here would be helpful, not sure I fully understand why this is needed